### PR TITLE
Another round of documentation cleanup

### PR DESCRIPTION
**Pull request by igorpecovnik**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -4,85 +4,101 @@ Linux for ARM development boards
...
+- Desktops - Install Desktop Environments
+- Netconfig - Network tools
+- DevTools - Development
+- Benchy - System benchmaking and diagnostics
+- Containers - Containerlization and Virtual Machines
+- Media - Media Servers and Editors
+- Management - Remote Management tools
 
-Due to lack of manpower we unfortunately have to be very picky about bug reports. Make sure to follow the bug reporting form available [here](https://armbian.com/bugs/) to learn how to collect necessary information and where to put your report depending on type of issue. Reports lacking fundamental diagnostics may be ignored.
+???+ success "Unit testing"
```
Comment (EvilOlaf) [inline docs/index.md:99]: are those question marks intended?

File: docs/User-Guide_Networking.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
-# Networking
+## Networking
 
 Armbian uses [**Netplan.io**](https://netplan.io/) to describe networking configurations. Netplan is a utility to easily configure Linux networking, using a declarative approach.
 If you want to configure your network manually, it is as simple as editing and creating Netplan yaml files (see the yaml configuration reference at the [Netplan docs](https://netplan.readthedocs.io/en/stable/netplan-yaml/)).
 
 Netplan is used to configure networks on **all** Armbian images since Release 24.05, no matter if minimal or desktop, Debian or Ubuntu. However, the networking backends are different based on if you choose a minimal image or not.
```
Comment (EvilOlaf) [inline docs/User-Guide_Networking.md:6]: It his correct? AFAIR minimal was systemd-networkd only?

Comment (igorpecovnik) [reply docs/User-Guide_Networking.md:6]: Minimal is by default systemd-networkd, yes. Perhaps changing to:

"Netplan is used to configure networks on **all** Armbian images, no matter if minimal or desktop, Debian or Ubuntu. However, the networking backend on minimal is `systemd-networkd` which has a very small footprint, runs much faster and thus contribute  to leaner user experience"

File: docs/index.md (hunk 1 above)
Comment (igorpecovnik) [reply docs/index.md:100]: Yes:

image

**Review by EvilOlaf — APPROVED** [review]

**Pull request merged**